packages/ionic/src/commands/package/package.ts

```diff
@@ -0,0 +1,270 @@
+import {
+  CommandLineInputs,
+  CommandLineOptions,
+  LOGGER_LEVELS,
+  OptionGroup,
+  contains,
+  validators
+} from '@ionic/cli-framework';
+import { columnar } from '@ionic/cli-framework/utils/format';
+import { sleep } from '@ionic/cli-framework/utils/process';
+import chalk from 'chalk';
+import * as Debug from 'debug';
+import * as fs from 'fs';
+import * as https from 'https';
+
+import { CommandMetadata } from '../../definitions';
+import { isSuperAgentError } from '../../guards';
+import { Command } from '../../lib/command';
+import { FatalException } from '../../lib/errors';
+
+import { fileUtils } from './utils';
+
+const debug = Debug('ionic:commands:package:build');
+const PLATFORMS = ['android', 'ios'];
+const BUILD_TYPES = ['debug', 'release', 'development', 'ad-hoc', 'app-store', 'enterprise'];
+
+interface PackageBuild {
+  job_id: number;
+  id: string;
+  caller_id: number;
+  platform: string;
+  build_type: string;
+  created: string;
+  finished: string;
+  state: string;
+  commit: any;
+  stack: any;
+  profile_tag: string;
+  automation_id: number;
+  environment_id: number;
+  native_config_id: number;
+  automation_name: string;
+  environment_name: string;
+  native_config_name: string;
+  job: any;
+}
+
+interface DownloadUrl {
+  url: string | null;
+}
+
+export class BuildCommand extends Command {
+  async getMetadata(): Promise<CommandMetadata> {
```

Comment:
If possible, this should have a description of what the command does. In the description, the base functionality and functionality changes via options have a chance of being explained in depth.

It looks like it will download the result, but I wouldn't know that by just reading the summary. I can guess what `--commit` does, but what does `--targetPlatform` do?

Reply:
will do